projects/angular/src/layout/nav/_header.clarity.scss

```diff
@@ -167,7 +194,12 @@
       flex-grow: 0;
       flex-shrink: 0;
       @include mixins.equilateral(header-variables.$clr-logo-width);
-      margin-right: variables.$clr_baselineRem_0_375;
+      @include mixins.css-var(
+        margin-right,
+        clr-header-branding-icon-gap,
+        variables.$clr_baselineRem_0_375,
+        variables.$clr-use-custom-properties
+      );
```

Comment:
The naming is intentional.
This will be a property that will remain part of the public API and after the token migration phase 2 (v17) it will be implemented with a gap on the parent. That's why I prefer to keep the real property's semantics and plan for future rework.
If I rename it, then I have to introduce a temporary -margin-right property, that will be removed later to only introduce the real one.